src/components/FloatingHelper/FloatingHelper.e2e.ts

```diff
@@ -12,19 +11,42 @@ describe('FloatingHelper', () => {
         await browser.get(storyUrl);
     });
 
-    eyes.it('should display title and text', async () => {
+    // TODO: autoExampleDriver.setProps() can not accept content: React.ReactNode prop,
+    // So we have to make another example for it. But it won't fit in the page, so we
+    // need to either scroll to it for the snapshot, or add ability to render a single example.
+    // We can also make a separate story for HelperContent and make it internal.
+    xit('should display title only', async () => {
```

Comment:
even after reading your TODO I am not sure why you skip the test

Reply:
I changed it so that all example variation exist in the storybook, and I run e2e eyes test on all.
there is no need to check if `title` exists or if `body` exists, anyways it's a css eyes manual verification.
(and existance of title, body, button is tested in ocmponent tests)